tests/test_testRepo.py

```diff
@@ -65,7 +65,8 @@ def tearDownClass(cls):
         removeTestTempDir(cls.root)
 
     def setUp(self):
-        self.butler = makeTestCollection(self.creatorButler)
+        # TestCase.id is unique for each test
```

Comment:
`id()` - otherwise it looks like a property/attribute to me.

It's unique for each "test method", but if you re-run with the same method it will be the same. Not sure if that needs to be made clear here.